docs/advanced_documentation/native-data-interface.md

```diff
@@ -137,9 +138,10 @@ One can import the `power_grid_meta_data` to get all the predefined `numpy.dtype
 The code below creates an array which is compatible with transformer input dataset.
 
 ```python
-from power_grid_model import power_grid_meta_data
+from power_grid_model import ComponentType, DataType, power_grid_meta_data
 
-transformer = np.empty(shape=5, dtype=power_grid_meta_data['input']['transformer']['dtype'])
+transformer = np.empty(shape=5, dtype=power_grid_meta_data[DataType.input][ComponentType.transformer]['dtype'])
+# transformer = np.empty(shape=5, dtype=power_grid_meta_data['input']['transformer']['dtype']) is supported as well.
```

Comment:
Remove commented out lines.

Reply:
@Jerry-Jinfeng-Guo I left the commented line on purpose to explain that the old format is still supported. It was a suggestion by @mgovers